From 101074ae4bb98533758289012819e72efbded021 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Sat, 22 Sep 2018 20:25:26 +0800
Subject: [PATCH 07/10] sunxi: video: let DE2 also use 6 as minimum dotclock
 divider

The user manual of A64 also uses 6 as the minimum dotclock divider, and
in experiments it's shown that 4 won't work.

Set the minimum dotclock divider to 6 when setting TCON_LCD dotclock for DE2.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 drivers/video/sunxi/lcdc.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/sunxi/lcdc.c b/drivers/video/sunxi/lcdc.c
index 2942e6244e..9f4a738c63 100644
--- a/drivers/video/sunxi/lcdc.c
+++ b/drivers/video/sunxi/lcdc.c
@@ -226,7 +226,7 @@ void lcdc_pll_set(struct sunxi_ccm_reg *ccm, int tcon, int dotclock,
 		max_m = 7;
 #endif
 #ifdef CONFIG_SUNXI_DE2
-		min_m = 4;
+		min_m = 6;
 		max_m = 127;
 #endif
 	} else {
-- 
2.18.1

